src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1516,6 +1516,14 @@ qlonglong TorrentImpl::nextAnnounce() const
     return lt::total_seconds(m_nativeStatus.next_announce);
 }
 
+static const qreal MONTH_SECONDS = 2629746;  // 1 month, as in C++20
+
+qreal TorrentImpl::popularity() const
+{
+    const qreal activeMonths = activeTime() / MONTH_SECONDS;
+    return activeMonths > 0 ? realRatio() / activeMonths : 0;
+}
```

Comment:
Fix code style:
```suggestion
qreal TorrentImpl::popularity() const
{
    const auto MONTH_SECONDS = std::chrono::duration_cast<std::chrono::seconds>(std::chrono::days(31)).count();
    const qreal activeMonths = activeTime() / MONTH_SECONDS;
    return (activeMonths > 0) ? (realRatio() / activeMonths) : 0;
}
```
or
```c++
qreal TorrentImpl::popularity() const
{
    const int MONTH_SECONDS = 2629746;
    const qreal activeMonths = activeTime() / MONTH_SECONDS;
    return (activeMonths > 0) ? (realRatio() / activeMonths) : 0;
}
```

Reply:
```c++
qreal TorrentImpl::popularity() const
{
    const int MONTH_SECONDS = 2629746;  // add some comments about how to calculate this number
    const qreal activeMonths = activeTime() / MONTH_SECONDS;
    return (activeMonths > 0) ? (realRatio() / activeMonths) : 0;
}
```
This solution won't work since `activeTime() / MONTH_SECONDS` produces integer values and only then those integers are converted to `qreal`. As the result, it will look like these:
  * 1st month: Popularity = 0
  * 2nd month: Popularity = Ratio
  * 3rd month: Popularity = Ratio / 2
  * 4th month: Popularity = Ration / 3
  * etc

<img src="https://github.com/qbittorrent/qBittorrent/assets/5607572/18bfffb7-af03-4b3c-8e07-a73899ab730a" width="40%" alt="screenshot">


Either `activeTime()` or `MONTH_SECONDS` should be `qreal` or casted to `qreal` before the division.